ui/common/css/header/_title.scss

```diff
@@ -8,21 +8,21 @@
   margin: 0 0.5rem;
   display: none;
 
-  @include breakpoint($mq-xx-small) {
+  @media (min-width: at-least($xx-small)) {
     display: block;
   }
 
-  @include breakpoint($mq-topnav-hidden) {
+  @media (max-width: at-most($medium)) {
```

Comment:
we are losing some meaning here, before it was clear that it was about the topnav being visible or not. But after the change it looks arbitrary, like it could be changed - but it could not.

In this this precise case I suppose we could use a mixin like the zoom one; but a bit later...